src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -1149,23 +1149,47 @@ public override BoundNode VisitDynamicInvocation(BoundDynamicInvocation node)
             return null;
         }
 
-        protected BoundNode VisitInterpolatedStringBase(BoundInterpolatedStringBase node)
+        protected BoundNode VisitInterpolatedStringBase(BoundInterpolatedStringBase node, InterpolatedStringHandlerData? nullableData)
```

Comment:
Do we need to account for some expressions being evaluated early due to argument attribute?

Reply:
Huh? That's not a thing.

Reply:
To be specific, they're not evaluated early. They're evaluated at exactly the same time as they were before.